libass/ass_fontselect.c

```diff
@@ -1001,7 +1000,7 @@ ass_embedded_fonts_add_provider(ASS_Library *lib, ASS_FontSelector *selector,
         load_fonts_from_dir(lib, lib->fonts_dir);
     }
 
-    for (i = 0; i < lib->num_fontdata; ++i)
+    for (size_t i = 0; i < lib->num_fontdata; ++i)
```

Comment:
While at it, also replace `++i` with `i++` as most developers prefer postfix form.

Reply:
Personally I prefer to only use the postfix form when the *"increment **after** evaluation"* part is required, but that's just a preference and I guess it's true that libass' for loops more often use `i++`.
Done.

Reply:
(FWIW I use prefix in my own non-libass code because it’s the standard type-agnostic default in C++, but indeed libass uses postfix.)